minifi-commons/minifi-commons-schema/src/test/java/org/apache/nifi/minifi/commons/schema/serialization/SchemaLoaderTest.java

```diff
@@ -84,13 +85,92 @@ public void testUnsupportedVersion() throws IOException, SchemaLoaderException {
         }
     }
 
+    @Test
+    public void testMinimalConfigV3VersionUnusedProperties() throws IOException, SchemaLoaderException {
+        Properties inputProperties = new Properties();
```

Comment:
Minor, but could we also have these tests drawing from a sample bootstrap.conf in lieu of just seeding the properties.  With some of the work that is underway surrounding C2 would not want to miss coverage of this functionality.